omod/src/main/webapp/fragments/field/concept.gsp

```diff
@@ -0,0 +1,41 @@
+<%
+	import groovy.json.*
```

Comment:
We don't use wildcard imports, and in general we try to avoid importing classes, I think you can build this json by hand